desktop-app/app/components/ErrorBoundary/index.js

```diff
@@ -1,4 +1,27 @@
 import React, {Fragment} from 'react';
+import TextAreaWithCopyButton from '../../utils/TextAreaWithCopyButton';
+import Button from '@material-ui/core/Button';
+
+const {openNewGitHubIssue} = require('electron-util');
+
+const reportBody = `Hi I'm reporting an app crash:
+<details>
+  <summary>Environment</summary>
+</details>
+<details>
+  <summary>Error Info</summary>
+</details>
+<details>
+  <summary>Stack Trace</summary>
+</details>`;
```

Comment:
@gorogoroumaru you need to fill those details sections
You have `Error Info` and `Stack Trace` in the `state`, wrap them in markdown code blocks using ` ``` `
merge `master` branch into this branch and use 
https://github.com/responsively-org/responsively-app/blob/3989fc2a9a19e7a52a79042a796c287899aedf46/desktop-app/app/utils/generalUtils.js#L19-L35
to fill the `Environment` section as we do in the [about dialog](https://github.com/responsively-org/responsively-app/blob/3989fc2a9a19e7a52a79042a796c287899aedf46/desktop-app/app/menu.js#L32)

Reply:
When you click the `Create Issue` button it should open the create issue form with all the info pre-filled, something like this:
https://github.com/responsively-org/responsively-app/pull/411#discussion_r479635731

Reply:
the user submitting the issue should replace `__PLEASE WRITE HERE ANY HINT THAT HELP US TO REPRODUCE THIS__` and put wherever she/he was doing when the app crashed

Reply:
Thanks, I have a trouble in fixing this.
When we call getEnvironmentInfo() in "desktop-app/app/components/ErrorBoundary/index.js", getPackageJson() is called.
This function is also called in "desktop-app/app/menu.js".
In the getPackageJson()'s implementation, it gets the directory path of its caller and calculate the package.json path by relative path.
"desktop-app/app/components/ErrorBoundary/index.js" and "desktop-app/app/menu.js" belong to different layer, so error occurs.
Please help.

Reply:
yeah that was a last minute change to help you develop this (maybe I did it wrong), I will take a look and come back to you ok?

Reply:
Awesome, thank you!

Reply:
gorogoroumaru can you go to Slack to talk?

Reply:
well @gorogoroumaru merge `master` and try again please